.gitlab-ci.yml

```diff
@@ -7,7 +7,7 @@ Pytest:
     # AK, 2020-12-13
     rm pytools/log.py
 
-    export EXTRA_INSTALL="numpy"
+    export EXTRA_INSTALL="numpy lark-parser"
```

Comment:
Could you push a sister MR to Gitlab (add the link to the PR description) to ensure that passes, too?

Reply:
https://gitlab.tiker.net/inducer/pytools/-/merge_requests/52